sorc/build_all.sh

```diff
@@ -9,6 +9,25 @@ set +x
 #                   Anything other than "true"  will use libraries locally.
 #------------------------------------
 
+_build_ufs_options=""
+
+while getopts "ac" option;do
+ case $option in
+  a)
+   echo "Received -a flag, building ufs-weather-model for ATMAERO app"
+   echo "skipping builds not needed for prototype runs"
+   _build_ufs_options=-a
+   break
+   ;;
+  c)
+   echo "Received -c flag, building ufs-weather-model for S2SW app"
+   echo "skipping builds not needed for prototype runs"
+   _build_ufs_options=-c
+   break
+   ;;
+ esac
+done
```

Comment:
Don't disagree. Right now this is just a shortcut for the APP to build. We could easily just change this to take the app as an argument instead of these switches. Although, hopefully in the near future we can build one app executable and use it for all configurations (at least until regional gets brought in or something, by which time this will be replaced).